server/client_manager.py

```diff
@@ -2107,3 +2110,33 @@ def get_multiclients(self, ipid=-1, hdid=""):
 
     def get_mods(self):
         return [c for c in self.clients if c.is_mod]
+        
+    class Battle_char:
+        def __init__(self, client, arg):
+            with open(
+                "storage/battlesystem/battlesystem.yaml", "r", encoding="utf-8"
+            ) as c:
+                char = yaml.safe_load(c)
+                self.fighter = arg
+                self.hp = float(char[arg]["HP"])
+                self.maxhp = self.hp
+                self.atk = float(char[arg]["ATK"])
+                self.defe = float(char[arg]["DEF"])
+                self.spa = float(char[arg]["SPA"])
+                self.spd = float(char[arg]["SPD"])
+                self.spe = float(char[arg]["SPE"])
+                self.target = None
+                self.selected_move = -1
+                self.status = None
+                self.moves = [ClientManager.Move(move) for move in char[arg]["Moves"]]
+
+    class Move:
+        def __init__(self, move):
+            self.name = move["Name"]
+            self.type = move["MovesType"]
+            self.power = float(move["Power"])
+            self.effect = move["Effects"]
+            tot = len(self.effect) * 50 + self.power
+            self.accuracy = -0.8 * tot + 156
+            if self.accuracy > 100:
+                self.accuracy = 100
```

Comment:
perhaps having a way to completely customize stats and formulas in the future by making them more abstract would help make this system more adaptable for any RPG format

Reply:
If you prefer, I could add accuracy to yaml file. So that everyone could customize it.
By the way, after this you want to merge it or you prefer to wait the whole code?

Reply:
Let's merge it when it's finished